src/algebra/group/hom_instances.lean

```diff
@@ -53,8 +56,51 @@ instance [add_zero_class M] [add_comm_monoid N] : add_comm_monoid (M →+ N) :=
 
 attribute [to_additive] monoid_hom.comm_monoid
 
+/-- If `G` is a commutative group, then `M →* G` a commutative group too. -/
```

Comment:
```suggestion
/-- If `G` is a commutative group, then `M →* G` is a commutative group too. -/
```

Reply:
I didn't even touch these comments, but why not :)